antismash/modules/clusterblast/core.py

```diff
@@ -233,12 +226,11 @@ def load_clusterblast_database(record: secmet.Record, searchtype: str = "cluster
                 a dictionary mapping cluster name to Cluster instance
                 a dictionary mapping protein name to Protein instance
     """
-    accessions = set()
+    names = set()
```

Comment:
As `load_reference_proteins()` doesn't need this anymore, it's unused and can be removed